CONTRIBUTING.md

```diff
@@ -75,8 +75,22 @@ In order to maintain a high software quality standard, we strongly prefer contri
 
 ## Coding Conventions
 
+Here are some rules of thumb in no particular order:
+
+- Do not use wildcard imports (with a few [exceptions](https://github.com/oss-review-toolkit/ort/blob/master/.detekt.yml)).
+- Sort references (like imports, dependencies etc.) strictly alphabetically (ASCII order, i.e. capital letters first),
+  with blank lines between imports from different top-level packages.
+- Group code logically using blank lines, e.g. before and after code blocks enclosed by curly braces, or before return
+  statements.
+- Do not use blank lines directly after opening curly braces.
```

Comment:
How about adding "Visually separate arrance, act, assert blocks by empty lines." as soft rule?

Reply:
I'd be fine with adding that rule explicitly, but I regard it simply as a a more concrete case of "Group code logically using blank lines". For now, I'll skip adding this in order to get the PR merged quickly.